tools/README.md

```diff
@@ -1,6 +1,35 @@
 AMS data tools
 ========================
 
+
+remove_orphan_kafka_topics
+------------------
+
+Remove orphan kafka topics is script that clears kafka from all orphan
+topics.Orphan topics are topics that do not exist in AMS; they have been
+left over on the kafka brokers.The script will check the existence of each
+kafka topic in the AMS data store.In case a topic isn't present in the AMS
+data store, the script will remove it from the broker.
+
+
+Requirements
+------------
+
+- kafka-python
+- pymongo
+
+How to run remove_orphan_kafka_topics
+-----------------------------
+`./remove_orphan_kafka_topics.py 
+--broker-list kafka1.host:9092,kafka2.host:9092,kafka3.host:9092
+ --mongo-host mongo.host --mongo-port 27017 --dry`
```

Comment:
This works for distributed mongo cluster as well? should we have the ability to add a list of mongo nodes here in case of cluster? or should we know ahead and target the PRIMARY node ?

Reply:
As of right now, we need to know the primary node in advance and target it.

Reply:
Can we easily modify it so the python script accepts the list of mongo servers and auto-selects the primary ? This should be easily done by the Mongo connection init in the driver